build.gradle

```diff
@@ -195,8 +214,8 @@ apply from: file('gradle/hacks/gradle-archives.gradle')
 
 apply from: file('gradle/hacks/wipe-temp.gradle')
 apply from: file('gradle/hacks/hashmapAssertions.gradle')
-apply from: file('gradle/hacks/turbocharge-jvm-opts.gradle')
-apply from: file('gradle/hacks/dummy-outputs.gradle')
+//apply from: file('gradle/hacks/turbocharge-jvm-opts.gradle')
+//apply from: file('gradle/hacks/dummy-outputs.gradle')
```

Comment:
Note that I had to disable some of the hacks because of some issues I was facing with them. Maybe this causes some gradle processes from the automation scripts to fail now?

Reply:
Hacks could be reenabled by simply replacing `.all { ... }` with `.forEach { ... }`.